### ISSUE TITLE
Add interactive UI mode, cf. vim-taskwarrior

### ISSUE THREAD

**Issue by linuxcaffe** [ledger/vim-ledger#18]
This is a gigantic feature request, but I think it would be awesome if (something like) vim-ledger were to behave something like vim-taskwarrior. 

https://github.com/farseer90718/vim-taskwarrior

Now it's true that taskwarrior is a task manager, and ledger is an accounting system, but they both have a similar approach with text-files and a rich command-set. Have a look, and perhaps you'll see what I mean.

**Comment by alerque** [comment]
I agree taskwarrior is cool, but I'm not sure how to bring that kind of interactive UI to this plugin ... or even entirely convinced that it _should_ be.

Have you seen @rcaputo's [vim-ledger_x](https://github.com/rcaputo/vim-ledger_x)?
